### DISCUSSION TITLE
Standardize covariates prior to fitting HMM?

### DISCUSSION THREAD

**Discussion by windstorm13** [bmcclintock/momentuHMM#82] [Q&A] [answered]
My initial analyses were in the moveHMM environment, which was suggested to standardize covariates prior to fitting the HMM.  

However since I have transitioned to working with momentuHMM, I have noticed that the momentuHMM tutorials do not standardize covariates prior to fitting the model, except for one occasion I found in the tutorial on page 59 when the boat distance covariate was standardized:
fulmar_data$boat.dist <- scale(fulmar_data$boat.dist)

Am I missing something here or is it not necessary to standardize the covariates prior to training in momentuHMM?

**Comment by bmcclintock** [comment, marked as answer]
This is a difficult statistical question that is not limited to momentuHMM or moveHMM. Others are of course welcome to chime in on the discussion here, but in my experience the decision to standardize, center, or otherwise rescale covariates must generally be made on a case-by-case basis. 

Neither moveHMM nor momentuHMM automatically standardize covariates, so this decision is entirely left to the user. There's lots of information (and differing opinions) on this topic out there (e.g. see this popular Cross Validated [discussion](https://stats.stackexchange.com/questions/29781/when-conducting-multiple-regression-when-should-you-center-your-predictor-varia) and a [paper](https://doi.org/10.1111/j.2041-210X.2010.00012.x) by Holger Schielzeth that is geared towards ecologists).

The only obvious case to me that is relevant to momentuHMM concerns covariates for the turn angle mean when using circular-circular regression. In this case, any angle covariates should most certainly not be standardized, centered, or otherwise rescaled in any way.
